### PR TITLE
DTGB-463: Added document block.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: gent_base.theme [hunk 1]
```diff
@@ -349,6 +351,47 @@ function gent_base_preprocess_field(&$variables) {
...
+    $url = file_create_url($file_entity->getFileUri());
+    $link_text = $file_entity->getFilename();
+
+    $variables['file_size'] = format_size($file->getSize());
+    $variables['file_type'] = strtr($mime_type, [
+      '/' => '-',
+    ]);
+
+    $options['attributes']['class'][] = 'standalone-link';
+    $options['attributes']['download'][] = 'download';
```
Comment (Jeroen005) [inline gent_base.theme:390]: Krijg je hier geen notice op? volgens mij moet dit gwn als string meegegeven worden.

File: gent_base.theme [hunk 2]
```diff
@@ -349,6 +351,47 @@ function gent_base_preprocess_field(&$variables) {
...
+
+    $variables['link'] = \Drupal::l($link_text, Url::fromUri($url, $options));
```
Comment (Jeroen005) [inline gent_base.theme]: l() is deprecated. Je moet de class https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Link.php/class/Link/8.2.x aanroepen

**Review by Jeroen005 — CHANGES_REQUESTED** [review]

File: gent_base.theme (hunk 2 above)
Comment (meirege) [reply gent_base.theme]: Fixed.

File: gent_base.theme (hunk 1 above)
Comment (meirege) [reply gent_base.theme:390]: Voor zover ik zie niet neen. Attribuut komt ook gewoon mooi door.

**Review by Jeroen005 — APPROVED** [review]

**Pull request merged**